### DISCUSSION TITLE
infinispan-tools:14.0.x versions not in Maven Central?

### DISCUSSION THREAD

**Discussion by davewichers** [infinispan/infinispan#11612] [General]
Per https://central.sonatype.com/artifact/org.infinispan/infinispan-tools/versions, no 14.0.x (non-Dev) versions of this library have been released to Central, like the 14.0.21.Final version, for example. And yet the project bom: https://repo1.maven.org/maven2/org/infinispan/infinispan-bom/14.0.21.Final/ specifically references that artifact.  Are they simply missing and need to be published? Or does that artifact not exist anymore and should have been removed from the bom?

**Comment by tristantarrant** [comment]
https://github.com/infinispan/infinispan/pull/11572

You need to wait for 14.0.22.Final

**Reply by davewichers** [reply]
OK. Are they going to also publish the 'missing' versions prior to that, for the people/projects using the older versions?

**Comment by tristantarrant** [comment]
I'm not sure the Sonatype ossrh maven publishing rules allow us to do that,
but I can try. Which version would you need?

**Reply by davewichers** [reply]
We need the current 14.0.21.Final version, but ideally you'd back publish them all. But if you could at least do that one, that would be great, so there is at least one version out there people can use for now. Thx

**Reply by tristantarrant** [reply]
I have released 14.0.21: https://repo1.maven.org/maven2/org/infinispan/infinispan-tools/14.0.21.Final/

**Comment by davewichers** [comment]
Awesome, thank you. If it's not too much trouble to do the others, that would be great. But not super important if it's a pain.
